hudi-flink/src/main/java/org/apache/hudi/table/format/mor/MergeOnReadInputFormat.java

```diff
@@ -431,6 +433,10 @@ public void close() {
     // iterator for log files
     private final Iterator<RowData> iterator;
 
+    // add the flag because the flink ParquetColumnarRowSplitReader is buggy:
```

Comment:
Can we file a Jira ticket to the Flink community and paste the jira id here so that we can track the progress of Flink.

Reply:
see https://issues.apache.org/jira/browse/FLINK-22370

Reply:
sounds good, can we add it to the comment of this PR(I mean this file.)?

Reply:
done